app/assets/src/components/common/DetailsSidebar/GeneDetailsMode/GeneDetailsMode.jsx

```diff
@@ -65,10 +67,12 @@ export default class GeneDetailsMode extends React.Component {
 
   async getGeneInfo(geneName) {
     const ontology = await getOntology(geneName);
-    const cardEntryFound = ontology.error !== "" ? false : true;
+    const ontologyInfoFound = ontology.error === "" ? true : false;
+    const cardEntryFound = ontology.label !== "" ? true : false;
```

Comment:
I noticed that you changed `label` to `drugClass` in `fetch_ontology` below. In that case, should this just be `cardEntryFound = ontology.drugClass !== ""`?

`cardEntryFound` looks like it's only used to decide whether to show `SOURCE_CARD`. Just wanted to confirm whether this was correct.

Reply:
Yes, that's correct.

Reply:
Awesome. To make the code clearer, I think it'd be helpful to add a couple of comments.

In `fetch_ontology`, maybe add a comment explaining that `drugClass` appears in every ontology (which is why you use it as a shortcut to determine whether there any fields to extract). Otherwise, future readers will be confused why you are using `drugClass` in particular.

I'm assuming that whenever `label` is in the ontology, so is `accession`. SOURCE_CARD seems to rely on `accession`. Could you change to `cardEntryFound = ontology.accession !== ""`? Otherwise you would want to add a comment explaining the relationship of `label` and `accession`.